specification/maps/data-plane/DataRegistry/preview/2022-12-01-preview/dataregistry.json

```diff
@@ -0,0 +1,660 @@
+        "dataFormat": {
+          "$ref": "#/definitions/DataFormat"
+        },
+        "connection": {
+          "$ref": "#/definitions/Connection"
+        },
+        "downloadURL": {
+          "description": "The download URL of the underlying data. It may not be available if the status of the data registry is `Running` or `Failed`. Note that `api-version` is not included.",
+          "type": "string",
+          "format": "uri",
+          "readOnly": true
+        },
+        "sizeInBytes": {
+          "description": "The size of the content in bytes.",
+          "type": "integer",
+          "format": "int64",
+          "readOnly": true
+        },
+        "contentMD5": {
+          "description": "The content MD5 of the content.",
+          "type": "string",
+          "readOnly": true
+        },
+        "status": {
+          "$ref": "#/definitions/LongRunningOperationStatus"
+        },
+        "error": {
+          "$ref": "../../../Common/preview/1.0/common.json#/definitions/MapsErrorDetail"
+        },
+        "registered": {
+          "description": "The registered timestamp.",
+          "type": "string",
+          "format": "date-time",
+          "readOnly": true
+        },
+        "updated": {
+          "description": "The updated timestamp.",
+          "type": "string",
+          "format": "date-time",
+          "readOnly": true
+        }
+      },
+      "required": [
+        "dataFormat",
+        "connection"
+      ]
+    },
+    "DataRegistryPatch": {
+      "description": "An object for patching a given data registry.",
+      "type": "object",
+      "properties": {
+        "description": {
+          "description": "The description to be given to the data registry. The max length is 1024 characters.",
+          "type": "string"
+        },
+        "dataFormat": {
+          "$ref": "#/definitions/DataFormat"
+        },
+        "connection": {
+          "$ref": "#/definitions/ConnectionPatch"
+        }
+      }
+    },
+    "Connection": {
+      "description": "A Connection object representing how Azure Maps Service accesses customer's resources and the scope of it.",
+      "type": "object",
+      "properties": {
+        "type": {
+          "description": "Connection type.",
+          "type": "string",
+          "enum": [
+            "AzureBlobFile"
+          ],
+          "x-ms-enum": {
+            "name": "ConnectionType",
+            "modelAsString": true,
+            "values": [
+              {
+                "value": "AzureBlobFile",
+                "description": "Connection to Azure Blob File."
+              }
+            ]
+          }
+        },
+        "msiClientId": {
+          "description": "ClientId of MSI. Use system assigned identity if not provided.",
+          "type": "string"
+        },
+        "linkedResource": {
+          "description": "The linked resource. Need to configure in control plane beforehand. Default is `default-linked-resource`",
```

Comment:
since we don't support default anymore. need to adjust the description.

Reply:
Fixed